modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -4823,3 +4823,309 @@ Description of the returned code::
 A variable-length relationship variable is bound more than once, which leads to no results because relationships must not occur more than once in each result. (Relationship r was repeated)
 ======
 =====
+
+// TODO label for Cypher25?
+[role=label--new-2024.12]
+[#_neo_clientnotification_cluster_servercaughtup]
+=== WAIT Server caught up
+
+.Notification category details
+[cols="<1s,<4"]
+|===
+|Neo4j  code
+m|Neo.ClientNotification.Cluster.ServerCaughtUp
+|Title
+a|Server is caught up.
+|Description
+a|Server `(%s)` at address `(%s)` is caught up.
+|Category
+m|GENERIC
+|GQLSTATUS code
+m|03N85
+|Status description
+a|
+info: server is caught up. Server `${ name }` at address `${ address }` is caught up.
+|Classification
+m|GENERIC
+|SeverityLevel
+m|INFORMATION
+|===
+
+
+.Successful completion of a `WAIT` command with two servers
+[.tabbed-example]
+=====
+[.include-with-GQLSTATUS-code]
+======
+Query::
++
+[source,cypher]
+----
+CREATE DATABASE foo TOPOLOGY 2 PRIMARIES WAIT
+----
+
+One notification is returned for each server in the cluster:
+
+Returned GQLSTATUS code::
+03N85
+
+Returned status description::
+info: server is caught up. Server `ServerId\{0e010000}` at address `localhost:20025` is caught up.
+
+Returned GQLSTATUS code::
+03N85
+
+Returned status description::
+info: server is caught up. Server `ServerId\{0e020000}` at address `localhost:20026` is caught up.
+======
+[.include-with-neo4j--code]
```

Comment:
@Lojjs this is assuming that you can get non-gql codes together with Cypher25? If not we should remove this section 🤷

Reply:
I haven't looked at the whole PR but the frameworks for errors and notifications should be independent of Cypher version. A specific query can of course give different notifications and errors in different Cypher versions, but the format is based on the Bolt version only so your assumption should be correct

Reply:
Thanks :)